webui/cypress/integration/configuration-file-page.spec.js

```diff
@@ -53,4 +54,18 @@ describe('Configuration file page', () => {
     cy.get('[data-cy="test_uploadZone"]').contains('Config upload failed: uploading system section ' +
       '"topology" is forbidden');
   });
+
+  it('Test: download config file', () => {
+
+    ///////////////////////////////////////////////////////////////////
+    cy.log('Download current conf file');
+    ////////////////////////////////////////////////////////////////////
+    cy.visit('/admin/cluster/configuration');
+    cy.downloadFile('http://localhost:8080/admin/cluster/config', 'cypress/downloads',
+      'config_downloaded.yml');
+    cy.readFile('cypress/downloads/config_downloaded.yml').should('contain',
+      '<!doctype html><html><head><title>Tarantool Cartridge</title><script>window.__tarantool_admin_prefix = "";' +
+      'window.__tarantool_variables = {"cartridge_refresh_interval":500,"cartridge_stat_period":2};' +
+      '</script></head><body><div id="root"></div>')
```

Comment:
It's not a config. The URL is wrong.

Reply:
Instead of hard-coding a link I suggest getting it dynamically. Try something like this:

```js
cy.get('[data-cy="downloadButton"]').should('have.attr', 'href')
    .then((href) => {
        cy.downloadFile(href)
    })
```

Reply:
Made changes, thank you